### PR TITLE
multi admin

### PR DESCRIPTION
**Pull request by zeroXten**
[Review on Reviewable](https://reviewable.io/reviews/pki-io/admin/70)

### CODE REVIEW

**Comment by jonbonazza** [comment]
Oh no! It doesn't pass CI. :x

**Comment by zeroXten** [comment]
Should now no?

**Comment by zeroXten** [comment]
The 1.4 build failed because of the EC error https://github.com/pki-io/core/issues/46

Restarting the build "fixed" it.

File: Makefile [hunk 1]
```diff
@@ -5,7 +5,7 @@ default: get-deps build test
 get-deps:
 	gom install
 build:
-	gom build pki.io.go adminApp.go runNode.go helpers.go runCA.go runOrg.go nodeApp.go runInit.go runPairingKey.go
+	gom build pki.io.go adminApp.go runNode.go helpers.go runCA.go runOrg.go nodeApp.go runInit.go runPairingKey.go runAdmin.go
```
Comment (jonbonazza) [inline Makefile]: Why do we have to explicitly list all files? Does gom not support building all files in the current module?

Comment (zeroXten) [inline Makefile]: Good point. https://github.com/pki-io/admin/pull/75

**Pull request merged**